.github/workflows/ci.yml

```diff
@@ -0,0 +1,67 @@
+name: CI
+
+on:
+  pull_request:
+    branches:
+      - master
+  push:
+    branches:
+      - master
+
+env:
+  RUSTFLAGS: -Dwarnings
+  RUST_BACKTRACE: 1
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Rust
+        run: rustup update stable
+      - run: cargo test --all-features --workspace
+
+  no-std:
+    strategy:
+      matrix:
+        target:
+          # TODO: Currently, valuable cannot build with targets that do not have atomic CAS.
+          #       We should port https://github.com/rust-lang/futures-rs/pull/2400.
+          # - thumbv6m-none-eabi
+          - thumbv7m-none-eabi
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Rust
+        run: rustup update stable
+      - run: rustup target add ${{ matrix.target }}
+      # TODO: Currently, valuable cannot build without `alloc` feature
+      #       because `Debug` impl of `dyn Enumerable` uses `format!` macro.
```

Comment:
This is mainly due to the following two lines.

https://github.com/tokio-rs/valuable/blob/7640b040d27f4db48bc5d97be47da71c659f4ee0/valuable/src/enumerable.rs#L109
https://github.com/tokio-rs/valuable/blob/7640b040d27f4db48bc5d97be47da71c659f4ee0/valuable/src/enumerable.rs#L124

@carllerche: Do you think it's ok to change it to print only variant name when alloc feature is disabled? (like the following)

```rust
#[cfg(feature = "alloc")]
let name = format!("{}::{}", self.name, variant.name());
#[cfg(not(feature = "alloc"))]
let name = variant.name();
```